pkgs/by-name/mu/multiqc/package.nix

```diff
@@ -0,0 +1,2 @@
+{python3Packages}:
+with python3Packages; toPythonApplication (multiqc.override {buildApplication = true;})
```

Comment:
What is this top-level `multiqc` attribute needed for?

Reply:
MultiQC is primarily used as a CLI app, but you can also import it as a Python package to use it in a Python script.

Just copied an example I found in fastapi-cli.

Reply:
> MultiQC is primarily used as a CLI app, but you can also import it as a Python package to use it in a Python script.
> 
> Just copied an example I found in fastapi-cli.

Hmm not justified IMO. The `buildApplication` argument does nothing, and the `toPythonApplication` is trivial - people can always use the original `python3Packages.multiqc` attribute.

Reply:
Discoverability? 

That does work yes, but the majority of users will use it as a CLI app, which is why I started it as a PR for that.

https://search.nixos.org/packages?channel=unstable&from=0&size=50&sort=relevance&type=packages&query=fastapi

Happy to cut it out if that's the final verdict, I'm just struggling to see whether this is an opinion or a requirement of nixpkgs.